source/tutorials/importing.rst

```diff
@@ -121,6 +124,89 @@ Importing data
      --input-path emp-paired-end-sequences \
      --output-path emp-paired-end-sequences.qza
 
+.. _`multiplexed barcode in seq import`:
+
+Multiplexed single-end FASTQ with barcode in sequence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Format description
+******************
+
+Users with multiplexed single-ended barcode in sequence reads should have:
+
+1. one ``fastq.gz`` file, containing records from multiple samples,
+2. one :doc:`metadata <metadata>` file with a column of per-sample barcodes for use in FASTQ demultiplexing
+
+Obtaining example data
+```````````````````````
+
+.. command-block::
+
+   mkdir muxed-se-barcode-in-seq
+
+.. download::
+   :url: https://data.qiime2.org/2020.5/tutorials/importing/muxed-se-barcode-in-seq.fastq.gz
+   :saveas: muxed-se-barcode-in-seq/sequences.fastq.gz
+
+Importing data
+**************
+
+.. command-block::
+
+   qiime tools import \
+     --type MultiplexedSingleEndBarcodeInSequence \
+     --input-path sequences.fastq.gz \
```

Comment:
This needs to be updated to match your download block above (or, you can alternatively keep this as-is and change the download block).

```suggestion
     --input-path muxed-se-barcode-in-seq/sequences.fastq.gz \
```